### ISSUE TITLE
Well! That is something out of the world!

### ISSUE THREAD

**Issue by mrfambo** [mantinedev/ui.mantine.dev#4]
Hey, will surely start being a sponsor of this amazing project. Wao! What an amazing work you have been doing and `ui.mantine.dev` is just being so much useful.

**Comment by deadcoder0904** [comment]
Agreed, looks like Tailwind UI got a new **real** competitor. How long it took to build the initial set of components?

**Comment by codart1** [comment]
I can't believe that awesome project got too low attention

**Comment by nirajkaushal** [comment]
This UI library is underrated...when I saw this library first time, I was shocked as to why I am not using this library. Thank you @rtivital for creating this awesome UI library.
